sdks/python/apache_beam/ml/inference/sklearn_inference.py

```diff
@@ -183,8 +167,8 @@ def run_inference(
       self,
       batch: Sequence[pandas.DataFrame],
       model: BaseEstimator,
-      inference_args: Optional[Dict[str, Any]] = None
-  ) -> Iterable[PredictionResult]:
+      inference_args: Optional[Dict[str, Any]] = None,
+      drop_example: bool = False) -> Iterable[PredictionResult]:
```

Comment:
Don't we need to pass this into _convert_to_result? Same question exists elsewhere too

Reply:
Done